react/graphql/getCollections.gql

```diff
@@ -0,0 +1,6 @@
+query GetCollection($fieldId: ID!) {
+  collection(id: $fieldId) {
```

Comment:
As a good practice, we should pass an annotation to the queries, like in [this example](https://github.com/vtex-apps/admin-catalog-translations/blob/ef3213c540de1a8219097c029d9343c0c90ac1c6/react/graphql/getProduct.gql#L2). This is helpful in two ways:
1. Tells the application the app we are trying to use when there are two queries out there with the same name;
2. Helps other developer to know the app we are querying - very helpful when debbuging.